elasticsearch-transport/lib/elasticsearch/transport/client.rb

```diff
@@ -125,6 +125,11 @@ def perform_request(method, path, params={}, body=nil)
         transport.perform_request method, path, params, body
       end
 
+      # Shuts down the client
+      def close
+        @transport.__close_connections
```

Comment:
Why is that wrong? Isn't that the point of private `__` methods? Shouldn't they be wrapped by more correct public ones. 

I think his is a pretty sane default impl.